tailcfg/tailcfg.go

```diff
@@ -2257,6 +2257,46 @@ type SSHRecordingAttempt struct {
 	FailureMessage string
 }
 
+// EnableFeatureRequest is a request sent to "/machine/f/{feature}/enable"
+// to enable a feature, such as Funnel, for the node's tailnet.
+//
+// This request itself DOES NOT modify tailnet configuration to enable the
+// feature, but returns information that can be presented to the user about
+// where/how to enable the feature. See EnableFeatureResponse for response
+// structure.
+type EnableFeatureRequest struct {
+	// NodeKey is the client's current node key.
+	NodeKey key.NodePublic `json:",omitempty"`
+}
```

Comment:
now that I see this written out, it's kinda weird that the feature string isn't in the request but lives outside in the URL.

I don't think REST-y purity should be our goal here: this is a weirdo API over noise only that end users can't really hit.  So the server endpoint could just be `/machine/feature` or something?

Reply:
Yeah - can do `/machine/feature` and move feature to the request